calculator_v5.rb

```diff
@@ -0,0 +1,93 @@
+# Goal is to create a program that takes two numbers and performs mathematical operations on them, then gives the result. Shooting for Gold level here.
+
+# outputs welcome string and user prompt
+puts "Welcome to Mathtropolis! Let's do some math!"
+puts "What type of math would you like to do?"
+
+# assigns user input to the variable "operation"
+operation = gets.chomp.downcase
+
+# defines a method called do_math that takes 3 arguments corresponding to
+# operation and inputted numbers below
+def do_math(operation, num1, num2)
+
+  # if operation is equal to values indicating addition
+  if operation == "+" || operation == "add" || operation == "addition" || operation == "adding"
```

Comment:
Ok, two notes about the comparisons in these `if`/`elsif` blocks.

First, if you have a long string of compound conditionals using `||` or `&&`, consider wrapping each clause in parens. Obviously your code runs and is technically correct, but `(operation == "+") || (operation == "add")` provides greater visibility into your intent/thought process, and can simplify debugging/refactoring later.

Second, you made great use of the `any?` and `include?` array methods below. Utilizing them in these conditionals would simplify the logic. Something like:

```ruby
adders = %w(+ add addition adding)
if adders.include?(operation) ...
```